man/man8/zfsprops.8

```diff
@@ -781,6 +782,35 @@ is equivalent to
 .Pc .
 .Pp
 The
+.Sy zstd
+compression using the zstd algorithm ,
+You can specify the
+.Sy zstd
+level by using the value
+.Sy zstd- Ns Em N ,
+where
+.Em N
+is an integer from 1
+.Pq fastest
+to 19
+.Pq best compression ratio .
+.Sy zstd
+You can also specify a negative 
+.Sy zstd
+level by using the value
+.Sy zstd- Ns Em N ,
+where
+.Em N
+is an integer from 100
+.Pq fastest
+to 1
+.Pq best compression ratio .
+.Sy zstd
+is equivalent to
+.Sy zstd-3
+.Pc .
+.Pp
+The
```

Comment:
This (part of) commit needs my signed-off-by

Reply:
thx, will add it